packages/client/src/components/Rules/Rules.tsx

```diff
@@ -1,56 +1,104 @@
-import { useNavigate } from 'react-router-dom';
-import { AppRoute } from 'utils/constants';
-import Button from 'components/Button';
-import { StLink, StFlex } from 'styles/global';
-import Card from 'components/Card';
+import Button from 'components/Button/Button';
+import Card from 'components/Card/Card';
+import useCarousel from 'utils/useCarousel';
+import RulesInfo from './RulesInfo';
+import { css } from 'styled-components';
+import {
+  StFlex,
+  StCarousel,
+  StCarouselBox,
+  width100Percent,
+} from 'styles/global';
 
 const cards = [
   {
     src: 'src/assets/img/start-game-1.jpg',
     alt: 'Карточка правил',
-    text: 'Избавляйтесь от карт одного цвета в первую очередь',
+    text: 'Избавляйтесь от\u00A0карт одного цвета в\u00A0первую очередь',
   },
   {
     src: 'src/assets/img/start-game-2.jpg',
     alt: 'Карточка правил',
-    text: 'Следите за ходами противника',
+    text: 'Следите за\u00A0ходами противника',
   },
   {
     src: 'src/assets/img/start-game-3.jpg',
     alt: 'Карточка правил',
-    text: 'Используйте карты действий в более подходящий момент',
+    text: 'Используйте карты действий в\u00A0более подходящий момент',
   },
 ];
 
+const TOTAL_SLIDES = 2;
+
+const buttonsPaddingWithIcon = css`
```

Comment:
можно этот миксин назвать нейтрально по отношению к компонентам и использовать дважды
```suggestion
const Padding7aliginItemsCenter = css`
  padding: 7px; //падинг для верха и низа задается так
  align-items: center;
```

Reply:
но на самом деле эта тенденция не очень по отношению к добавлению нескольких свойств
может все таки использовать отдельный компонент?
либо можно создать в асетах/стили миксин файл и туда прописывать миксины и там же их комбинировать

Reply:
``` padding: 7px; ``` задаст отступ 7 со всех сторон и перетрет отступы слева/справа